settingService/blockUser.go

```diff
@@ -106,12 +99,11 @@ func IsUserBlocked(byteRequest [] byte) (isBlocked int, err error) {
 		loger.Log.Errorf("Error has occurred: ", err)
 		return nil, err
 	}
-	isBlocked = result.IsBlocked
-	if isBlocked == 0 {
-		return 0, nil
+	if result.IsBlocked == 0 {
+		return false, nil
 	}
-	if isBlocked == 1 {
-		return 1, nil
+	if result.IsBlocked == 1 {
+		return true, nil
 	}
 	err = errors.New("Function has failed")
 	return nil, err
```

Comment:
Fixed. In a case of failed function will return true, which means that message mustn't be sent.